.github/workflows/deployment.yml

```diff
@@ -0,0 +1,176 @@
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '7.0.x'
+
+    - name: Set the correct value to PACKAGE_DEPLOYMENT_BRANCH
+      shel: pwsh
+      run: |
+        if [[ "$WORKFLOW_EVENT_NAME" == "workflow_call" ]]; then \
+          echo "${BRANCH_NAME_REQUESTED_BY_CALL}" >> ${PACKAGE_DEPLOYMENT_BRANCH} \
+          echo "${BRANCH_NAME_REQUESTED_BY_CALL,,}" >>${PACKAGE_DEPLOYMENT_BRANCH_LOWERCASE} \
+          echo "${BRANCH_NAME_REQUESTED_BY_CALL}" >> ${CONFIGURATION} \
+        if [[ "$WORKFLOW_EVENT_NAME" == "workflow_dispatch" ]]; then \
+          echo "${BRANCH_NAME_REQUESTED_BY_DISPATCH}" >> ${PACKAGE_DEPLOYMENT_BRANCH} \
+          echo "${BRANCH_NAME_REQUESTED_BY_DISPATCH,,}" >>${PACKAGE_DEPLOYMENT_BRANCH_LOWERCASE}
+          echo "${BRANCH_NAME_REQUESTED_BY_DISPATCH}" >> ${CONFIGURATION} \
+      env:
+        WORKFLOW_EVENT_NAME: ${{ github.event_name }} 
+        BRANCH_NAME_REQUESTED_BY_CALL: ${{ workflow_call.inputs.branch }}
+        BRANCH_NAME_REQUESTED_BY_DISPATCH: ${{ workflow_dispatch.inputs.branch }}
+
+    - name: Configure the package manifest & logo sets
+      shell: pwsh
+      run: |
+        . './scripts/Configure-AppxManifest.ps1' `
+          -Branch "$env:PACKAGE_DEPLOYMENT_BRANCH" `
+          -PackageProjectDir "$env:PACKAGE_PROJECT_DIR" `
+          -Publisher "$env:SIDELOAD_PUBLISHER" `
+          -WorkingDir "$env:WORKING_DIR"
+      env:
+        SIDELOAD_PUBLISHER: ${{ secrets.SIDELOAD_PUBLISHER_SECRET }}
+
+    - name: Inject the Bing Maps API token
+      shell: pwsh
+      run: |
+        . './scripts/Replace-SpecialString.ps1' `
+          -WorkingDir "$env:WORKING_DIR" -Target "bingmapskey.secret" -New "$env:BING_MAPS_SECRET"
+      env:
+        BING_MAPS_SECRET: ${{ secrets.BING_MAPS_SECRET }}
+
+    - name: Inject the AppCenter token
+      shell: pwsh
+      run: |
+        . './scripts/Replace-SpecialString.ps1' `
+          -WorkingDir "$env:WORKING_DIR" -Target "appcenter.secret" -New "$env:APP_CENTER_SECRET"
+      env:
+        APP_CENTER_SECRET: ${{ secrets.APP_CENTER_SECRET }}
+
+    - name: Inject the GitHub OAuth client ID
+      run: |
+        . './scripts/Replace-SpecialString.ps1' `
+          -WorkingDir "$env:WORKING_DIR" -Target "githubclientid.secret" -New "$env:GH_OAUTH_CLIENT_ID"
+      env:
+        GH_OAUTH_CLIENT_ID: ${{ secrets.GH_OAUTH_CLIENT_ID }}
+
+    - name: Use Windows SDK Preview
+      shell: cmd
+      run: |
+        for /f %%a in ('dir /b /a:d %localappdata%\Microsoft\VisualStudio\17*') do echo UsePreviews=True>%localappdata%\Microsoft\VisualStudio\%%a\sdk.txt
+
+    - name: Restore NuGet
+      shell: pwsh
+      run: 'nuget restore $env:SOLUTION_NAME'
+
+    - name: Restore Files.sln
+      shell: pwsh
+      run: |
+        msbuild $env:SOLUTION_NAME `
+        -t:Restore `
+        -p:Platform=$env:PLATFORM `
+        -p:Configuration=$env:CONFIGURATION `
+        -p:PublishReadyToRun=true
+
+    - name: Build & package Files
```

Comment:
The store build has a different list of build args

Reply:
wdym? This PR is for stable sideload. Store submission still should be ran with Azure Pipeline.

Reply:
I saw some references to the Store build in `Configure-AppxManifest.ps1` but if we're not doing that one yet, you can disregard my comments.

Reply:
Yeah it is just preperation.